### ISSUE TITLE
Sequence Diagram bidirectional arrows are not supported

### ISSUE THREAD

**Issue by Foshkey** [mjbvz/vscode-markdown-mermaid#278]
Bidirectional arrows were [added in v11.0.0](https://mermaid.js.org/syntax/sequenceDiagram.html#messages). However, this doesn't appear to render correctly in the preview.

### Example

```mermaid
sequenceDiagram
    participant Alice
    participant Bob
    Bob<<->>Alice: Shakes hands
```

### How it's rendered in VS Code

<img width="594" alt="image" src="https://github.com/user-attachments/assets/9d4094f0-9079-4065-bc3d-ea9d2e035f20">

**Comment by Foshkey** [comment]
Ope, I see there's already [a PR](https://github.com/mjbvz/vscode-markdown-mermaid/pull/277) for this 😅

**Comment by mjbvz** [comment]
Should be fixed by #277
